CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
     Changes that have landed in master but are not yet released.
     Click to see more.
   </summary>
+
+  Minor: Add Section component
```

Comment:
For copy pasta purposes, we have been prefixing these changelog items with `-`. But it's been hard to maintain this particular section and I'm. thinking maybe it does not matter. 🤔

Reply:
It's all good, I'll make the change and adjust in the future.